### DISCUSSION TITLE
support for netNamedPipeBindingSync

### DISCUSSION THREAD

**Discussion by kharivarma** [CoreWCF/CoreWCF#763] [Q&A]
I am trying to migrate an existing WCF service (net framework 4.8) to .Net 6.

The service is exposed thorough netNamedPipeBindingSync protocol. 

Can anyone please let me know if CoreWCF supports netNamedPipeBindingSync protocol otherwise can I use netTcpBinding in place of netNamedPipeBindingSync ?

**Comment by TrabacchinLuigi** [comment]
AFAIK core WCF does not still support named pipe, but it will. Have you tried the prerelease/beta packages ?
I've tried to migrate one of my WCF project to the released version. One with named pipe also, but i wasn't able to comunicate with it, just to build it...
